core/src/main/java/bisq/core/btc/setup/WalletsSetup.java

```diff
@@ -522,6 +523,16 @@ public boolean isDownloadComplete() {
         return downloadPercentageProperty().get() == 1d;
     }
 
+    public boolean isChainHeightSyncedToPeers() {
```

Comment:
Maybe better rename it so it reflects that we have a tolernace for the check... 
Maybe: `inSyncWithMostCommonChainHeight` ? (still not great but the "MostCommon" suggests we are not operating on exact data here.

Reply:
Done!  renamed to `isChainHeightSyncedWithinTolerance`